api-report/tree.api.md

```diff
@@ -1035,6 +1037,11 @@ export function singleTextCursor(root: JsonableTree): ITreeCursorSynchronous;
 // @alpha
 type Skip = number;
 
+// @internal
```

Comment:
Is it okay that we're exporting an internal? I can see someone making changes down the line mistakenly thinking it won't break consumers. That said I can't really imagine why we'd want to change these definitions or how doing so would break consumers.